codec/protobuf/protobuf.go

```diff
@@ -18,7 +19,8 @@ type protobufCodec int
 func (c protobufCodec) Encode(v interface{}) ([]byte, error) {
```

Comment:
I agree about the fix here, but in this case the implementation differs from the interface definition (it is not a pure protobuf encode  anymore), so a proper GoDoc on the Encode method would be nice.